src/modules/translation/languages/en.json

```
@@ -580,20 +580,41 @@
     "table": {
       "name": "Name",
       "link": "Link",
-      "assigned_to": "Assigned To"
+      "assigned_to": "Assigned To",
+      "address": "Address",
+      "alias": "Alias",
+      "land": "Land",
+      "actions": "Actions",
+      "owner": "Owner"
     },
     "assigned_to_land": "Land ({landId})",
     "assigned_to_estate": "Estate ({landId})",
     "button": {
       "assign": "Use as Link",
       "edit": "Edit Link",
-      "use_as_alias": "Use as Alias"
+      "use_as_alias": "Use as Alias",
+      "add_to_avatar": "Add to Avatar",
+      "link_to_address": "Link to an Ethereum Address",
+      "assign_to_land": "Assign to Land o WORLDs"
```

I think the name can be assigned to only one World, right?
```suggestion
      "assign_to_land": "Assign to Land or World"
```